lib/lita/handlers/locker_misc.rb

```diff
@@ -48,9 +48,18 @@ def log(response)
 
       def status(response)
         name = response.match_data['label']
-        return response.reply(status_label(name)) if Label.exists?(name)
-        return response.reply(status_resource(name)) if Resource.exists?(name)
-        response.reply(failed(t('subject.does_not_exist', name: name)))
+        unless name =~ /\*/
```

Comment:
At some point I suspect this logic will want to get broken out beyond just status (`lock prod*` for example), but this seems OK for now.  I usually try to avoid an assignment in a conditional check, but that's just because I've gotten bit by it in the past.